.golangci.yaml

```diff
@@ -2,6 +2,7 @@
 
 linters:
   disable:
+    - contextcheck
```

Comment:
⛏️ Can you please add this to `.golangci.next.yaml` also? One of its gripes seems wrong and one seems right.